README.md

```diff
@@ -1,99 +1,128 @@
-# parg
+# argparse
 ## Project Status
-[ ![Codeship Status for clagraff/parg](https://codeship.com/projects/68eb7800-af6b-0133-1b97-3e80188314d9/status?branch=master)](https://codeship.com/projects/132507)
-[![GoDoc](https://godoc.org/github.com/clagraff/parg?status.svg)](https://godoc.org/github.com/clagraff/parg)
+[ ![Codeship Status for clagraff/argparse](https://codeship.com/projects/68eb7800-af6b-0133-1b97-3e80188314d9/status?branch=master)](https://codeship.com/projects/132507)
+[![GoDoc](https://godoc.org/github.com/clagraff/argparse?status.svg)](https://godoc.org/github.com/clagraff/argparse)
 
 ## Description
-A Golang flag & argument parser for program arguments. The goal of parg (program-arguments) is to emulate the usability and functionallity of Python's [argparse](https://docs.python.org/dev/howto/argparse.html#the-basics) package, where setting up a parser and arguments is both easy and trivial.
+A Golang flag & argument parser for program arguments. The goal of argparse (program-arguments) is to emulate the usability and functionallity of Python's [argparse](https://docs.python.org/dev/howto/argparse.html#the-basics) package, where setting up a parser and arguments is both easy and trivial.
 
 # Install
 Installing is simple, as with most other Golang packages:
 
 ```bash
-$ go get github.com/clagraff/pargs
+$ go get github.com/clagraff/argparse
 ```
 
 Boom! All set! Feel free to read on for examples on getting started and using this package.
 
 # The Basics
 ## Create a parser
-Imagine we have a basic program which outputs text to the user depending on which flags/arguments they provide. First, we will need to setup a parser:
+Imagine we have a basic program which outputs text to the user depending on which flags/arguments they provide. We need create a parse, create and include our various options,
```

Comment:
Spelling errors: `create a parse`
Should be: `create a parser`

Also, might be good to improve this description.